### ISSUE TITLE
HTML rendering

### ISSUE THREAD

**Issue by volodymyrdovhan** [WisdmLabs/moodle-auth_edwiserbridge#34]
Hello,
it is better to move HTML generation and rendering to mustache templates:
![image](https://github.com/user-attachments/assets/78b93896-9bab-43bb-9217-326b8591271a)

**Comment by jaydeeptrivedi13** [comment]
@volodymyrdovhan Yes, you are correct. We have moved this code to a mustache template file.
